x/superfluid/keeper/migrate_test.go

```diff
@@ -616,6 +629,73 @@ func (s *KeeperTestSuite) TestMigrateNonSuperfluidLockBalancerToConcentrated() {
 	}
 }
 
+func (s *KeeperTestSuite) TestMigrateUnlockedPositionFromBalancerToConcentrated() {
+	defaultJoinTime := s.Ctx.BlockTime()
+	type sendTest struct {
+		unlocking                bool
+		percentOfSharesToMigrate sdk.Dec
+		tokenOutMins             sdk.Coins
+		expectedError            error
+	}
+	testCases := map[string]sendTest{
+		"no lock (full shares)": {
+			percentOfSharesToMigrate: sdk.MustNewDecFromStr("1"),
```

Comment:
Would be great if we can also add cases for:
1. `percentOfSharesToMigrate ` > 1 (assuming this should error)
2. `percentOfSharesToMigrate` = 0

Reply:
added extra test cases here https://github.com/osmosis-labs/osmosis/pull/5360/commits/d95a5bae6da27bb73a7208c01870a4cd9bd5cf4e